src/wwwroot/template-editor/adapterService.js

```diff
@@ -29,8 +29,7 @@
       getPreviewImageWithPlay: getPreviewImageWithPlay,
       getSettings: getSettings,
       refreshMercadoShopsProductServiceToken: refreshMercadoShopsProductServiceToken,
-      getTiendaNubeProducts: getTiendaNubeProducts,
-      overrideUploadFiles: overrideUploadFiles
```

Comment:
Is removing this necessary? I mean, _overrideUploadFiles_ seems to be related to uploadings and your changes seems to be related to getting the gallery.

So, we do not know yet if we will need to override some behavior in file uploading.